Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
+
+COPY lib/AI-Agent.xml /opt/app/
+COPY build/libs/bulk-scan-end-to-end-tests.jar /opt/app/
+
+CMD [ "bulk-scan-end-to-end-tests.jar" ]
```

Comment:
do we need this ?

Reply:
we don't need this but pipeline mandates to have Dockerfile

Reply:
I meant those 3 lines